netutils/config/parser.py

```diff
@@ -949,3 +949,77 @@ def _build_nested_config(self, line):
                 self.indent_level = spaces
 
             self._update_config_lines(line)
+
+
+class NokiaConfigParser(BaseSpaceConfigParser):
+    """Nokia SrOS config parser."""
+
+    comment_chars = ["# ", "#--------------------------------------------------"]
+    banner_start = []
+
+    def __init__(self, config):
+        """Create ConfigParser Object.
+
+        Args:
+            config (str): The config text to parse.
+        """
+        super(NokiaConfigParser, self).__init__(config)
+
+    def is_exit_next(self, line):  # pylint: disable=no-self-use
+        """Determine if line has 'exit' or 'exit all' in it.
+
+        Args:
+            line (str): A config line from the device.
+
+        Returns:
+            bool: True if line has 'exit' or 'exit all', else False.
+        """
+        if line.lstrip() == "exit" or line.lstrip() == "exit all":
+            return True
+        return False
+
+    def is_section_title(self, line):  # pylint: disable=no-self-use
```

Comment:
I think is_section_title and get_section_title should be private methods by convention, e.g. `_is_section_title`

Reply:
Updated to be private.